apps/ledger-live-mobile/src/screens/Platform/index.tsx

```diff
@@ -18,6 +18,7 @@ export function Catalog(props: CatalogProps) {
 export function LiveApp(props: LiveAppProps) {
   const config = useFeature("discover");
 
+  // TODO: both version looks the same now
   return config?.enabled && config?.params.version === "2" ? (
     <LiveApp2 {...props} />
   ) : (
```

Comment:
I don't see a diff between LiveApp1 and LiveApp2. TODO: Check if it's fine to ditch 🤔